File: CPP/Ruko.cpp
#pragma once
#include <iostream>
#include <string>


using namespace std;


class Ruko
{
private:
    string namaUsaha;
    string alamat;
    string jenisBisnis;
public:
    Ruko()
    {


    }


    Ruko(string namaUsaha, string alamat, string jenisBisnis)
    {
        this->namaUsaha = namaUsaha;
        this->alamat = alamat;
        this->jenisBisnis = jenisBisnis;
    }

    // Setter methods
    void setNamaUsaha(string namaUsaha) {
        this->namaUsaha = namaUsaha;
    }
    
    void setAlamat(string alamat) {
        this->alamat = alamat;
    }
    
    void setJenisBisnis(string jenisBisnis) {
        this->jenisBisnis = jenisBisnis;
    }
    
    // Getter methods
    string getNamaUsaha() {
        return this->namaUsaha;
    }
    
    string getAlamat() {
        return this->alamat;
    }
    
    string getJenisBisnis() {
        return this->jenisBisnis;
    }


    ~Ruko()
    {


    }
};

File: PY/Ruko.py
class Ruko:
    __namaUsaha = ""
    __alamat = ""
    __jenisBisnis = ""
    
    def __init__(self, namaUsaha="", alamat="", jenisBisnis=""):
        self.__namaUsaha = namaUsaha
        self.__alamat = alamat
        self.__jenisBisnis = jenisBisnis
    
    # Setter methods
    def setNamaUsaha(self, namaUsaha):
        self.__namaUsaha = namaUsaha
    
    def setAlamat(self, alamat):
        self.__alamat = alamat
    
    def setJenisBisnis(self, jenisBisnis):
        self.__jenisBisnis = jenisBisnis
    
    # Getter methods
    def getNamaUsaha(self):
        return self.__namaUsaha
    
    def getAlamat(self):
        return self.__alamat
    
    def getJenisBisnis(self):
        return self.__jenisBisnis
